.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,3 +7,11 @@ Fixes #
   -
 
 ## Any background context you want to provide?
+
+## Checklist
+
+Not all points below apply to all pull requests.
+
+- [ ] I have added a new feature and have added tests to go along with it.
+- [ ] I have fixed a bug and have added a regression test.
+- [ ] I have added a test using random numbers; I have made sure it uses a seed, and that random numbers generated are valid inputs for the tests.
```

Comment:
NIce!

Reply:
Note, codacy warns about these checkboxes as it's an extension of github flavoured markdown, but given that the template is only for use here on github I'm going to ignore the warnings.